lib/intelligent_foods.rb

```diff
@@ -6,6 +6,9 @@
 require "ostruct"
 
 require "intelligent_foods/api_client"
+require "intelligent_foods/api_authorization"
+require "intelligent_foods/basic_authorization"
+require "intelligent_foods/bearer_authorization"
```

Comment:
Since we're using the `authorization` name multiple times, it may make sense to put these into an authorizations folder and organize them that way